### ISSUE TITLE
java.lang.NullPointerException: Attempt to invoke virtual method 'int android.support.v7.widget.LinearLayoutManager.findLastCompletelyVisibleItemPosition()' on a null object reference

### ISSUE THREAD

**Issue by benobab** [dimelo/Dimelo-Android#31]
Hi,

I'm getting this error in my Firebase Crash Reporting dashboard, it's causing a Fatal error.

```
Exception java.lang.NullPointerException: Attempt to invoke virtual method 'int android.support.v7.widget.LinearLayoutManager.findLastCompletelyVisibleItemPosition()' on a null object reference
com.dimelo.dimelosdk.main.Chat.onSaveInstanceState (Chat.java:726)
android.support.v4.app.Fragment.performSaveInstanceState (Fragment.java:2418)
android.support.v4.app.FragmentManagerImpl.saveFragmentBasicState (FragmentManager.java:2661)
android.support.v4.app.FragmentManagerImpl.saveAllState (FragmentManager.java:2732)
android.support.v4.app.Fragment.performSaveInstanceState (Fragment.java:2420)
android.support.v4.app.FragmentManagerImpl.saveFragmentBasicState (FragmentManager.java:2661)
android.support.v4.app.FragmentManagerImpl.saveAllState (FragmentManager.java:2732)
android.support.v4.app.FragmentController.saveAllState (FragmentController.java:125)
android.support.v4.app.FragmentActivity.onSaveInstanceState (FragmentActivity.java:571)
android.support.v7.app.AppCompatActivity.onSaveInstanceState (AppCompatActivity.java:509)
android.app.Activity.performSaveInstanceState (Activity.java:1502)
android.app.Instrumentation.callActivityOnSaveInstanceState (Instrumentation.java:1308)
android.app.ActivityThread.callCallActivityOnSaveInstanceState (ActivityThread.java:4964)
android.app.ActivityThread.performStopActivityInner (ActivityThread.java:4242)
android.app.ActivityThread.handleStopActivity (ActivityThread.java:4301)
android.app.ActivityThread.-wrap28 (ActivityThread.java)
android.app.ActivityThread$H.handleMessage (ActivityThread.java:1663)
android.os.Handler.dispatchMessage (Handler.java:102)
android.os.Looper.loop (Looper.java:154)
android.app.ActivityThread.main (ActivityThread.java:6692)
java.lang.reflect.Method.invoke (Method.java)
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:1468)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:1358)
```

**Android version :**
- 7.0
- 6.0
- 6.0.1

I'm using your sdk v**1.6.3**

Thanks for your help.

**Comment by jarthod** [comment]
Hi, thanks for reporting, do you know any way to reproduce this issue? do you have any numbers on the frequency of this crash? Also could you please give us the first 5 numbers of your API key?

**Comment by benobab** [comment]
Hi, thanks for your reply. 57 users got this error for now (it's a small percentage - 1% approx).

API_KEY beginning with AIzaS
GCM key beginning with 87616
SECRET beginning with ef5fc

**Comment by jarthod** [comment]
Thanks, we'll have a look and keep you posted

**Comment by waelba** [comment]
Hi @benobab ,
Do you know any way to reproduce this issue ?
Thanks

**Comment by benobab** [comment]
Unfortunately, I don't :/ I tried a few times but the bug does not appear on my phones.

It could seem dumb, but can't you just add a check for this view in Chat.java ? To avoid findLastCompletelyVisibleItemPosition() to cause a nullPointerException?

Since it's appearing on less than 1% of our users, it could just avoid them to get a crash ^^

Let me know if I can help.

**Comment by waelba** [comment]
Ok thanks, we'll get back to you.

**Comment by benobab** [comment]
@waelbenabdallah hi! Any news related to this issue?

**Comment by waelba** [comment]
Hi @benobab,
sorry for the wait it's fixed in `v1.6.5`
Thanks
